src/include/platform/ThreadStackManager.h

```diff
@@ -104,6 +104,7 @@ class ThreadStackManager
     CHIP_ERROR GetPrimary802154MACAddress(uint8_t * buf);
     CHIP_ERROR GetExternalIPv6Address(chip::Inet::IPAddress & addr);
     CHIP_ERROR GetPollPeriod(uint32_t & buf);
+    CHIP_ERROR SetTxPower(uint8_t txPower);
```

Comment:
I would expect adding this API also to ConnectivityManager and related files, similarly to other Set/Get Thread methods.